api/v1/report.proto

```diff
@@ -18,131 +18,18 @@ package istio.mixer.v1;
 
 option go_package="istio.io/mixer/api/v1;mixerpb";
 
-import "google/protobuf/timestamp.proto";
-import "google/protobuf/struct.proto";
-import "google/protobuf/any.proto";
+import "attributes.proto";
 import "google/rpc/status.proto";
 
 // Used to report telemetry after performing an action.
 message ReportRequest {
   // Uniquely identifies this request, used to match to responses
   int64 request_id = 1;
 
-  // The set of discovered facts associated with this request. This value
```

Comment:
I've reworked the proposal a bit, added back aggregation with timespans, and removed the well known attributes.

At this point we're almost back to your proposal, minus logs, and with metrics grouped in with attributes.

We _could_ break metrics out, but I don't really see the need, it complicates the api for no real benefit. This API is really just a "report a set of attributes over some time periods" API, and not a general purpose monitoring API.